Clock.py

```diff
@@ -164,11 +171,11 @@ def announce(s, kob, sender):
         print("Time announcement interval must be betwen 1 and 1440.")
         sys.exit(1)
     annc_interval = args.Interval * 60      # announcement interval (sec)
-    local_text = args.Text;
+    local_text = args.Text
     
     myKOB = kob.KOB(port=port, audio=sound)
-    mySender = morse.Sender(speed)
-    
+    mySender = morse.Sender(text_speed)
+     
```

Comment:
Is there a blank space at the start of line 178 (which is otherwise empty)?

Reply:
Yes. Thanks for catching that. I removed it.